### ISSUE TITLE
Excessive number of dog, cat and fish species

### ISSUE THREAD

**Issue by Coolthulhu** [cataclysmbnteam/Cataclysm-BN#864]
All critters should differ "significantly enough" to warrant getting an extra type. Cats, dogs and fish have many near-identical subtypes.

The question is whether to use generic names like "big dog" (medium size), "small dog" (small size), "big fish" (medium size) etc. or pick representative species out of each group and have the whole continent dominated by say, husky dogs and salmon.

**Comment by DrPariah** [comment]
Personally I like it, it adds color and life to the world. I'd just rather that they spawned more often (I've only seen a single Maine Coon spawn in a year plus of playing CDDA and BN combined). Are you looking at this as a baseline, so we could just move the specific species to modded content? Like, say Generic Animals are the default and you can enable the species as a mod for those folks who like their Chonker Cats?

**Comment by Coolthulhu** [comment]
> Are you looking at this as a baseline, so we could just move the specific species to modded content?

Yes. Having a mod that adds animals with different names is fine.
Core stuff is better off with all differences being meaningful.
